doc/introduction/data.rst

```diff
@@ -101,18 +101,27 @@ Viewing Available Datasets
 --------------------------
 
 We can call the 
-:func:`~pennylane.data.list_datasets` function to get a snapshot of the currently available data.
+:func:`~pennylane.data.list_data_names` function to get a snapshot of the currently available data.
 This function returns a nested dictionary as shown below. 
```

Comment:
```suggestion
:func:`~pennylane.data.list_data_names` function to get a snapshot of the currently available data.
This function returns a list of strings as shown below. 
```
Why are we changing the name of the function here? `list_datasets` sounded better IMO.

Reply:
This is a separate function to the `list_datasets()` function which is currently removed in this PR. The docs here were repurposed for the `list_data_names()` which is a separate function, used internally for the interactive loading of datasets, but also has some practical value as a public facing function.